linera-views/src/memory.rs

```diff
@@ -201,11 +290,8 @@ impl<E> MemoryContext<E> {
             cache_size: 1000,
         };
         let config = MemoryStoreConfig { common_config };
-        let namespace = "linera";
-        let store = MemoryStore::connect(&config, namespace)
-            .now_or_never()
-            .unwrap()
-            .unwrap();
+        let namespace = generate_test_namespace();
```

Comment:
Why this?

Reply:
If we keep using the "linera" then calling twice creates a collision.
That makes the function utility quite limited.